data/org.freedesktop.UDisks2.policy.in

```diff
@@ -588,6 +588,27 @@
     </defaults>
   </action>
 
+  <!-- Host NQN and Host ID manipulation -->
+  <action id="org.freedesktop.udisks2.nvme-set-hostnqn">
```

Comment:
Do we need separate policies for these two? I guess it doesn't hurt, but we might consider using just a single policy to keep the file shorter and simpler :)

Reply:
Right, I've merged those two.